data/ecosystem/vendors.yaml

```diff
@@ -356,3 +356,10 @@
   contact: 'https://github.com/hagen1778'
   oss: true
   commercial: true
+- name: Tracetest
+  distribution: false
+  nativeOTLP: true
+  url: 'https://tracetest.io/'
+  contact: 'https://tracetest.io/contact/'
```

Comment:
please provide an email address or github handle. This is for our purpose to reach out in any case we want to rework that page or the link is broken